cloud/aws/runtime/secret/secrets_manager.go

```diff
@@ -153,14 +153,20 @@ func (s *secretsManagerSecretService) Access(ctx context.Context, sv *secret.Sec
 		)
 	}
 
+	returnValue := result.SecretBinary
+
+	if returnValue == nil && result.SecretString != nil {
+		returnValue = []byte(*result.SecretString)
+	}
+
```

Comment:
Needed this fix to support other secret data types that were used outside of the scope of the nitric API.